ci/check_spider_naming_consistency.py

```diff
@@ -0,0 +1,263 @@
+import ast
+import os
+import re
+import sys
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from typing import List, Tuple
+
+COUNTRYCODE_COMPONENTS = {
+    "AE",
+    "AL",
+    "AR",
+    "AT",
+    "AU",
+    "BE",
+    "BG",
+    "BH",
+    "BO",
+    "BR",
+    "BY",
+    "CA",
+    "CH",
+    "CL",
+    "CN",
+    "CY",
+    "CZ",
+    "DE",
+    "DK",
+    "DO",
+    "EE",
+    "EG",
+    "ES",
+    "EU",
+    "FI",
+    "FJ",
+    "FR",
+    "GB",
+    "GG",
+    "GR",
+    "GT",
+    "HK",
+    "HR",
+    "HU",
+    "ID",
+    "IE",
+    "IL",
+    "IM",
+    "IN",
+    "IT",
+    "JE",
+    "JP",
+    "KE",
+    "KR",
+    "KW",
+    "KZ",
+    "LT",
+    "LU",
+    "LV",
+    "MA",
+    "ME",
+    "MK",
+    "MO",
+    "MT",
+    "MX",
+    "MY",
+    "NI",
+    "NL",
+    "NO",
+    "NZ",
+    "OM",
+    "PE",
+    "PH",
+    "PK",
+    "PL",
+    "PR",
+    "PT",
+    "QA",
+    "RO",
+    "RS",
+    "RU",
+    "SA",
+    "SE",
+    "SG",
+    "SI",
+    "SK",
+    "TH",
+    "TR",
+    "TW",
+    "UA",
+    "UK",
```

Comment:
UK isn't a real country code. I guess this comes from "HSBC UK". Slight preference for `HsbcUkGBSpider`, since "HSBC UK" is the brand for GB.

Reply:
Thanks, I thought it was for something else but that was the only one that broke when I removed it so away it goes!